neutron/values.yaml

```diff
@@ -207,6 +207,15 @@ pg_metrics:
         - gauge:
             usage: "GAUGE"
             description: "Internet IPs used per subnet"
+    openstack_neutron_monitor_sec_grp:
+     query: "select securitygroupportbindings.security_group_id, count(*) as count from securitygroupportbindings join securitygrouprules on securitygroupportbindings.security_group_id = securitygrouprules.security_group_id group by securitygroupportbindings.security_group_id order by count desc limit 10"
+     metrics:
+        - security_group_id:
+            usage: "LABEL"
+            description: "security group id details"
+      - count:
+            usage: "GAUGE"
+            description: "Count of Sec Grp ID"              
```

Comment:
Could you change the description of "count" to  "Estimate on number of filter rules",
and could you double check that `security_group_id` is sufficent as id?
I fear, it requires also a labelling in the query with `AS security_group_id`.